src/dex/synthetix/synthetix.ts

```diff
@@ -89,13 +89,13 @@ export class Synthetix extends SimpleExchange implements IDex<SynthetixData> {
 
   async initializePricing(blockNumber: number) {
     if (this.synthetixState.onchainConfigValuesWithUndefined === undefined) {
-      await this.synthetixState.updateOnchainConfigValues(blockNumber);
+      await this.synthetixState.updateOnchainConfigValues('latest');
```

Comment:
I am not sure this is correct. In initialize pricing we must respect passed blockNumber. Otherwise, we can not test pricing for specific block number

Reply:
Initializing with latest remove all possible failure

Reply:
For this we have retry mechanism. Or either we should have some flag to identify that it is testing phase

Reply:
@Verisana updated